Tests/TensorFlowTests/LossTests.swift

```diff
@@ -67,6 +67,18 @@ final class LossTests: XCTestCase {
 
         assertElementsEqual(expected: expectedGradients, actual: gradients)
     }
+    
+    func testNegativeLogLikelihood() {
+        let predicted = Tensor<Float>(shape: [2, 4], scalars: [1, 2, 3, 4, 5, 6, 7, 8])
```

Comment:
Can you also have an expected variable which just checks if the outputs are right? You can add the Assert equal to the same test case.